package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "badgen-cli",
+  "version": "0.0.1",
+  "description": "",
+  "keywords": [
+    "cli",
+    "badge",
+    "svg"
+  ],
+  "bin": {
+    "badgen": "cli.js"
+  },
+  "scripts": {
+    "lint": "standart",
```

Comment:
I think we better use a broadly adopted linter, why not just `standard` 😉

Reply:
OMG, that typo squatting!

Reply:
😂